.github/workflows/deploy.yml

```diff
@@ -35,11 +32,11 @@ jobs:
 
       - name: Run Forge anvil
         run: |
-          anvil --fork-url ${{ secrets.RPC_URL }} &
+          anvil --fork-url https://rpc.ankr.com/eth &
           sleep 10
         id: anvil
 
       - name: Run Forge scripts
         run: |
-          forge script --broadcast --rpc-url ${{ secrets.LOCAL_RPC_URL }} --private-key ${{ secrets.PRIVATE_KEY }} --sig 'run(string)' script/DeployAll.s.sol:DeployAll script/deployParameters/local.json
+          forge script --broadcast --rpc-url http://127.0.0.1:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --sig 'run(string)' script/DeployAll.s.sol:DeployAll script/deployParameters/local.json
```

Comment:
Would it be better if we use environment variables here and save the values in .env?

Reply:
I don't think I've ever seen a project with a `.env` file in the root directory, usually there's only a `.env.sample`. Additionally, I'd like to treat these environment variables as specific to these yml files, rather than as shared environment variables for the entire repo.